static/index.html

```diff
@@ -27,6 +27,19 @@ <h1 class="mb-5">Auth0 SPA JS Playground</h1>
 
       <div v-if="!loading">
         <div class="btn-toolbar justify-content-between">
+          <div class="btn-group mb-3">
+            <button class="btn btn-info" @click="useAuth0" data-cy="use-auth0">
+              Use Auth0
+            </button>
+
+            <button
+              class="btn btn-info"
+              @click="useNodeOidcProvider"
+              data-cy="use-node-oidc-provider"
+            >
+              Use Node OIDC Provider
+            </button>
+          </div>
```

Comment:
This allows toggling between Auth0 and Node OIDC. The latter is mostly for the automated tests whereas Auth0 will be used when manually interacting with the playground.